.github/workflows/test.yml

```diff
@@ -620,6 +620,12 @@ jobs:
             - name: Rebuild
               run: pnpm rebuild
             - name: Test
+              env:
+                  COVERAGE_THRESHOLD_LINES: 99.41
+                  COVERAGE_THRESHOLD_FUNCTIONS: 100
+                  COVERAGE_THRESHOLD_STATEMENTS: 99.43
+                  COVERAGE_THRESHOLD_BRANCHES: 99.18
```

Comment:
@ItsANameToo reduced this because it is related to the logic of rendering PendingReview or Success, which depends on the polygon address monitoring. We can restore this once we have the functionality implemented so that we can add the necessary mocks.

Reply:
alright